doc/man3/CRYPTO_SIGNAL_block.pod

```diff
@@ -0,0 +1,68 @@
+=pod
+
+=head1 NAME
+
+CRYPTO_SIGNAL_block, CRYPTO_SIGNAL_block_set,
+CRYPTO_SIGNAL_unblock_all - Signal support for OpenSSL
+
+=head1 SYNOPSIS
+
+ #include <openssl/crypto.h>
+
+ int CRYPTO_SIGNAL_block_set(CRYPTO_SIGNAL **props);
+ int CRYPTO_SIGNAL_block(CRYPTO_SIGNAL *p);
+ int CRYPTO_SIGNAL_unblock_all(void);
+
+=head1 DESCRIPTION
+
+OpenSSL provides basic facilities to work with signals, especially aimed
+to handle signals while using multi-threaded routines. The following
+functions are provided:
+
+=over 2
+
+=item *
+
+To set a handler for a specific or a set of signals, CRYPTO_SIGNAL_block()
+or CRYPTO_SIGNAL_block_set() may be used, respectively.
```

Comment:
It's unclear from the description. I assume the `set` in the name is referring to a set of signals. If so, how do we know how big the set is?

Reply:
It expected a null terminated array. Added an explicit size_t argument, docs changed accordingly.